src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -283,7 +283,7 @@ test('should fire split panel resize event', () => {
     />
   );
   wrapper.findSplitPanel()!.findSlider()!.keydown(KeyCode.pageUp);
-  expect(onSplitPanelResize).toHaveBeenCalledWith({ size: 460 });
+  expect(onSplitPanelResize).toHaveBeenCalled();
```

Comment:
Because I changed `style.height` to `clientHeight`, this no longer grabs the size. There is an integration test that tests that the width is actually changing how we expect.